Source/Core/Common/x64Emitter.h

```diff
@@ -440,13 +440,19 @@ class XEmitter
   void PUSHF();
   void POPF();
 
+  enum class Force5Bytes
+  {
+    Yes,
+    No,
+  };
```

Comment:
Someone in the future is bound to find this and wonder why this isn't a simple boolean. A comment explaining this, might be wise.

Reply:
This is a common pattern in our code base that does not need a comment.